velox/expression/ExprToSubfieldFilter.cpp

```diff
@@ -512,6 +512,13 @@ std::unique_ptr<common::Filter> leafCallToSubfieldFilter(
       }
       return isNull();
     }
+  } else if (call.name() == "isnotnull") {
```

Comment:
Isn't `IS NOT NULL` parsed into `not(is_null(...))`?

Reply:
In Spark there is an expression 'IsNotNull' and it is frequently used in filter pushdown.This issue adds the background for this change: https://github.com/facebookincubator/velox/issues/11093. Thanks.